Integrations/integration-RTIR.yml

```diff
@@ -192,6 +218,14 @@ script:
             'EntryContext': ec
         })
 
+    def ticket_request(ticket_id):
```

Comment:
improve function name, like `search_ticket` and the other one to be `search_ticket_request`